pyiron_contrib/atomistic/atomicrex/structure_list.py

```diff
@@ -150,7 +150,8 @@ def from_hdf(self, hdf=None, group_name="arstructure"):
             structure = Atoms()
             structure.from_hdf(hdf_s_lst, group_name="structure")
             self.structure = structure
-            self.fit_properties = hdf_s_lst["fit_properties"]
+            self.fit_properties = ARFitPropertyList()
+            self.fit_properties.from_hdf(hdf=hdf_s_lst, group_name="fit_properties")
```

Comment:
This `from_hdf` fails because `ARFitPropertyList` doesn't accept a container anymore in its `__init__`, which is something the `InputList` code (reasonably) assumes all its subclasses can.  Will look into a work-around after dinner.

Reply:
That is fixed in https://github.com/pyiron/pyiron_contrib/pull/72/commits/ed54ffe188d3dd24470bf4c1c806da5d06692f9c